src/lib/paginator/paginator.spec.ts

```diff
@@ -0,0 +1,303 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {MdPaginatorModule} from './index';
+import {MdPaginator, PageEvent} from './paginator';
+import {Component, ElementRef, ViewChild} from '@angular/core';
+import {MdCommonModule} from '../core';
+import {MdSelectModule} from '../select/index';
+import {MdTooltipModule} from '../tooltip/index';
+import {MdButtonModule} from '../button/index';
+import {MdPaginatorIntl} from './paginator-intl';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {customMatchers} from '../core/testing/jasmine-matchers';
+import {dispatchMouseEvent} from '../core/testing/dispatch-events';
+
+
+describe('MdPaginator', () => {
+  let fixture: ComponentFixture<MdPaginatorApp>;
+  let component: MdPaginatorApp;
+  let paginator: MdPaginator;
+
+  beforeEach(async(() => {
+    jasmine.addMatchers(customMatchers);
+
+    TestBed.configureTestingModule({
+      imports: [
+        MdButtonModule,
+        MdCommonModule,
+        MdSelectModule,
+        MdTooltipModule,
+        MdPaginatorModule,
```

Comment:
Gah, made the same mistake twice. Thanks for pointing this out, much better to mimic what the users will do